__tests__/app.test.js

```diff
@@ -91,6 +91,86 @@ describe("GET /api/reviews", () =>{
             expect(body.msg).toBe("404 not found")
         })
     })
+    describe(" GET /api/reviews (queries) - Feature Enhancement", () => {      
+        test("Should return 11 reviews only for the social deduction category ", () =>{
+            return request(app)
+        .get("/api/reviews?category=social deduction")
+        .expect(200)
+        .then(({ body }) => {
+            const { reviews } = body
+            expect(reviews).toHaveLength(11)
+            
+            reviews.forEach(category => {
+                expect(category).toHaveProperty("comment_count", expect.any(String))
+                expect(category).toHaveProperty("owner", expect.any(String))
+                expect(category).toHaveProperty("title", expect.any(String))
+                expect(category).toHaveProperty("review_id", expect.any(Number))
+                expect(category).toHaveProperty("category", expect.any(String))
```

Comment:
Would you expect any category here? You need to look specifically for social deduction to make sure this query has worked.